app/model/exchange/passmarksettings/PassMarkSettings.scala

```diff
@@ -52,3 +52,16 @@ object Phase2PassMarkSettings {
   implicit val phase2PassMarkSettingsFormat = Json.format[Phase2PassMarkSettings]
   implicit val phase2PassMarkSettingsHandler = Macros.handler[Phase2PassMarkSettings]
 }
+
+case class Phase3PassMarkSettings(
+  schemes: List[Phase3PassMark],
```

Comment:
schemes is a bit misleading agreed, but given that this is a field associated with Phase3PassMarkSettings there shouldn't be too much of confusion. Also i want to keep it the same way as other phases